src/main/java/io/anserini/search/query/CARTitleTopicReader.java

```diff
@@ -0,0 +1,59 @@
+package io.anserini.search.query;
```

Comment:
What is the difference between `CarTitleTopicReader.java` and `CarTopicReader.java`?
I think we should have only one of them?

Reply:
CAR provides two files for query: title and topic. They are in different formats so I write different topic reader. Might combine them in one Reader class?

Reply:
I think so. 
Remember topic is represented in `SortedMap<String, Map<String, String>>`? 
The key of the inner Map is the _tag_ of the value (actual query) of the inner Map.
For example, ```{"1": {"title": "university of waterloo", "description": "where is university of waterloo", ...}, ...}```